apps/user-office-backend/src/datasources/postgres/AdminDataSource.ts

```diff
@@ -91,18 +88,13 @@ export default class PostgresAdminDataSource implements AdminDataSource {
       .returning('*');
 
     if (!institutionRecord) {
-      throw new Error('Could not create call');
+      return null;
```

Comment:
We now return now here unless we no longer need throwing an error looks fine

Reply:
@mutambaraf thanks for noticing this one. I think the change here is that I moved the error handling in the mutation/logic layer instead of database layer. It is here: https://github.com/UserOfficeProject/user-office-core/pull/133/files/71e7ca9c5da4db73ed7aa0dc9a857d8d6b31a534?diff=split&w=1#diff-61a948f290a5811fde75287846cdda53de34db50b47afc1e29a001ed642c85e9R107

Because PR is quite big with lot of changes you might need to wait a bit until this link opens at the right line.